examples/asr/transcribe_speech.py

```diff
@@ -116,7 +116,7 @@
 class ModelChangeConfig:
 
     # Sub-config for changes specific to the Conformer Encoder
-    conformer: ConformerChangeConfig = ConformerChangeConfig()
+    conformer: ConformerChangeConfig = field(default_factory=ConformerChangeConfig)
```

Comment:
Needs () here

Reply:
Do you mean I have to pass a lambda to `default_factory`? isn't it redundant in this case?

Reply:
I think it should be OK as-is, the dataclass's constructor works as a factory here.

Reply:
I dunno if I'm mixing two things but I had errors at runtime if I used just the class and not the object. let's be safe and use object

Reply:
@titu1994 The code works just fine for me. I was able to transcribe a small dataset successfully. It was only after I tried making the changes you requested that I got errors.

Reply:
Alright then